walrus/tests.py

```diff
@@ -54,6 +54,12 @@ class Stat(BaseModel):
 class HashModel(BaseModel):
     data = HashField()
 
+class DefaultOption(BaseModel):
+    default_empty = JSONField()
+    default_dict = JSONField(default={})
```

Comment:
You wouldn't want your default value to be mutable. This goes along with not using mutable objects as default function parameters.